### DISCUSSION TITLE
Problems with finding saved files after crash

### DISCUSSION THREAD

**Discussion by TricksterSam** [plugdata-team/plugdata#1424] [Q&A]
I'm a bit confused, because I've been saving frequently my, main patch and abstractions, but I just had a crash and none of my recent files are actual recent saves. They all are older versions. Any idea what could be wrong? Where are the autosaves stored? Is there a reason plugdata might not be saving when I click Save?

**Comment by TricksterSam** [comment]
If there was a second copy of the patch open in the background when plugdata crashed, could it have autosaved over what I had been saving?

**Comment by timothyschoen** [comment]
Hi, sorry this happened! I definitely want to find out what is causing this. What version of plugdata and what OS are you using?

**Comment by TricksterSam** [comment]
Thanks, Timothy!

I'm using plugdata 0.8.3 in PopOS 22.04.

Luckily, I remember a lot of my changes, and now that I'm rewriting code,
I'm discovering other fixes I need to make.

Sam
